### PR TITLE
Introduce simple tooltip

### PR DESCRIPTION
**Pull request by thomasjo**
In order to make it easier to "hit" the wrap guide region in order to display the tooltip, I made the element wider, and pushed the actual line into an `:after` pseudo-element. Instead of the original `1px` width, the wrap guide is now `5px` wide. I think that should be sufficient.

:rotating_light: Do not merge this before merging #39. :rotating_light: 

/cc @atom/feedback

### CODE REVIEW

File: lib/wrap-guide-element.coffee [hunk 1]
```diff
@@ -89,6 +90,9 @@ class WrapGuideElement extends HTMLDivElement
     else
       @style.display = 'none'
 
+  setTooltip: ->
+    @title = "This is the wrap guide."
```
Comment (benogle) [inline lib/wrap-guide-element.coffee]: Maybe "This line is the wrap-guide. You can toggle it with cmd-ctrl-shift-omg-w"

\* No idea on the keybinding for toggling this. Maybe we should tell them to search the command palette

Comment (lee-dohm) [inline lib/wrap-guide-element.coffee]: I was just thinking of tooltip text of "wrap-guide" and trusting people to search the command palette. But yeah, it doesn't cost us anything to go all the way :grinning:

File: styles/wrap-guide.less [hunk 2]
```diff
@@ -3,10 +3,22 @@
 atom-text-editor::shadow {
   .wrap-guide {
     height: 100%;
-    width: 1px;
+    width: 5px;
     z-index: 3;
     position: absolute;
     top: 0;
-    background-color: @syntax-wrap-guide-color;
+    background-color: transparent;
```
Comment (lee-dohm) [inline styles/wrap-guide.less]: Will this do crazy stuff to subpixel anti-aliasing?

Comment (thomasjo) [inline styles/wrap-guide.less]: Not sure. How should I go about checking if it does? What's the current workaround if it does? Using RGBA or HSLA with the alpha channel set to 0?

**Comment by thomasjo** [comment]
Added a (very) simple test. Any other reasonable test ideas?
Otherwise I think this might be ready to :boat:

File: styles/wrap-guide.less (hunk 2 above)
Comment (simurai) [inline styles/wrap-guide.less]: Are you on OS X? You could use the zoom feature https://www.youtube.com/watch?v=jJV7CMgW5Dg to see it closer. "Smooth images" turned off and I like "Picture-in-picture".

Or take a screenshot and then zoom in:

screen shot 2015-07-24 at 5 35 17 pm

If you see the text edges having blue on the left and red on the right, then subpixel anti-aliasing works. So in this case it looks fine.

File: lib/wrap-guide-element.coffee [hunk 3]
```diff
@@ -89,6 +90,9 @@ class WrapGuideElement extends HTMLDivElement
     else
       @style.display = 'none'
 
+  setTooltip: ->
+    @title = "This line is the wrap-guide. You can quickly toggle it via the command palette."
```
Comment (benogle) [inline lib/wrap-guide-element.coffee]: I wonder if people are going to know what to do with this. Something more explicit would be nice. Something short that tells them what the command palette is or how to invoke might be nice. 

> Toggle it by invoking the command palette (`cmd-shit-p`), and searching for wrap-guide.

They only need to know that stuff once though.

**Comment by thomasjo** [comment]
Updated the tooltip to refer to *View* menu instead of command palette. This obviously depends on #39 where the command is introduced.

**Pull request closed without merging**